xblockutils/helpers.py

```diff
@@ -0,0 +1,17 @@
+"""
+Useful helper methods
+"""
+
+
+def child_isinstance(block, child_id, block_class_or_mixin):
+    """
+    Is "block"'s child identified by usage_id "child_id" an instance of
+    "block_class_or_mixin"?
```

Comment:
This could be written better. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings